cmd/validate/image.go

```diff
@@ -151,6 +152,11 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 
 			  ec validate image --image registry/name:tag --output yaml --output appstudio=<path>
 
+			Write output the data used in policy evaluation as YAML file
```

Comment:
This sentence reads oddly to me. Maybe pick either "write" or "output" but not both?

Reply:
Yeah, how about: `Write the data used in the policy evaluation to a file in YAML format`